finch/src/main/scala/caliban/FinchHttpAdapter.scala

```diff
@@ -0,0 +1,42 @@
+package caliban
+
+import caliban.Value.NullValue
+import io.circe.Json
+import io.circe.syntax._
+import io.finch._
+import io.finch.circe._
+import zio.interop.catz._
+import zio.{ Runtime, Task, URIO }
+
+object FinchHttpAdapter extends Endpoint.Module[Task] {
```

Comment:
This is also super minor but how about naming this `FinchAdapter`, to be consistent with `Http4sAdapter`. The Akka one has `Http` in it because the library itself is named `Akka Http`.